src/dotnet/commands/dotnet-new/CSharp_Lib/project.json.template

```diff
@@ -0,0 +1,9 @@
+{
+  "version": "1.0.0-*",
+  "dependencies": {
+    "NETStandard.Library": "1.5.0-rc3-24126-00"
```

Comment:
FYI - We are switching this to version 1.6 for RTM. See https://github.com/dotnet/corefx/pull/8925 cc @ericstj